packages/drivers/odsp-driver-definitions/src/factory.ts

```diff
@@ -141,4 +141,10 @@ export interface HostStoragePolicy {
 	 * where 'createLinkType' was requested.)
 	 */
 	enableSingleRequestForShareLinkWithCreate?: boolean;
+
+	/**
+	 * True if host does not want the storage service to use the prefetch cache to get the snapshot. Undefined will be treated
+	 * as false. This is if the host wants to do some A/B testing.
+	 */
+	avoidPrefetchSnapshotCache?: boolean;
```

Comment:
Is this useful setting to expose? I.e. what would be a scenario?
I feel like you are adding it to be able to disable new code in case of an issue. And if so, is it better to use feature gate (kill-bit switch) instead? It's more directly telling the intention.

Reply:
No, it is not for that. Cosmin mentioned that he wants to have some hostpolicy in case they want to do a good A/B testing for performance like when they call prefetch api and then load the container, then how fast is to load if we use the prefetch promise from cache and when we are not. So, they can write their logic and use this policy to achieve that. 
We don't need a feature gate because if some error occurred while prefetching then, according to current logic I am simply just proceeding with the network call according to old flow. 
Once we have more confidence then I can just throw if the prefetch fails and not make a network call.